pkg/crypto/tinkcrypto/primitive/jwe/ecdhes/register_ecdhes_aead_helper.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package ecdhes
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/google/tink/go/core/registry"
+	gcmpb "github.com/google/tink/proto/aes_gcm_go_proto"
+	tinkpb "github.com/google/tink/proto/tink_go_proto"
+	"github.com/square/go-jose/v3"
+
+	"github.com/hyperledger/aries-framework-go/pkg/crypto/tinkcrypto/primitive/jwe/subtle/ecdhes"
+)
+
+const (
+	// nolint:gochecknoglobals
+	aesGCMTypeURL = "type.googleapis.com/google.crypto.tink.AesGcmKey"
```

Comment:
I assume this one is actually defined in upstream tink?

Reply:
Yes, it's not exported, so I had to copy it here.